src/frontend/Elaborator.ml

```diff
@@ -202,6 +205,21 @@ and chk_tm : CS.con -> T.Chk.tac =
         | D.TpDim, _, _ -> RM.ret @@ R.Dim.literal n
         | _ -> RM.ret @@ R.Nat.literal n
       end
+      
+    | CS.D0 ->
```

Comment:
Yeah, I had that originally, but thought the syntax overloading could be confusing for the user in the future where they appear in the same line of code (i.e. in a cofibration). I don't care strongly about it, so I can change it back if we prefer.

Reply:
Personally, I think we should default to respecting the "bidirectional" discipline and always overload constructors, because it makes the code simpler. But I do see what you are saying about usability issues; I think we could find out if it causes usability issues once we can write down some examples.

But I don't feel so strongly about it, it's just a thought.

Reply:
This is why you have 0️⃣ and 1️⃣